docs/learn-launch.md

```diff
@@ -0,0 +1,83 @@
+---
+id: learn-launch
+title: Polkadot Launch Timeline
+sidebar_label: Polkadot Launch Timeline
+description: An explanation of Polkadot's launch process and its individual phases
+---
+
+The Polkadot network will have a phased roll-out plan, with important milestones toward
+decentralization marking each phase. Up until the final phase, the chain will be considered a _chain
+candidate_ (CC), and not fully launched. During the CC phase, the chain may reset and change
+dramatically due to critical bugs, important changes in the code, or other circumstances.
+
+## The PoA Launch
+
+A PoA (Proof of Authority) chain is a permissioned type of blockchain in which a set of pre-approved
+entities produce blocks. There is no game theory or [incentivization](learn-staking), no competition
+between the authority nodes and no [randomness](learn-randomness). The authorities take turns in
+producing blocks for the chain. During this phase, the Web3 Foundation will be managing all six
+authorities.
+
+Even though there will be no community-run [validators](maintain-validator) securing the network at
+this point, potential validators can start joining the network and signaling their intention to
+participate in [consensus](learn-consensus).
+
+No other functionality will be enabled. It will not be possible to transfer tokens, register
+[identities](learn-identity), create [democracy](maintain-guides-democracy) or
+[treasury](learn-treasury) proposals, lease [parachain](learn-parachains) slots, or send
+[cross-chain messages](learn-crosschain).
+
+## PoA to NPoS
+
+Once Web3 Foundation is confident in the stability of the network and there are a sufficient number
+of validator intentions, Web3 Foundation will use [Sudo](https://youtu.be/InekMjJpVdo) - a superuser
+account with access to [governance](learn-governance) functions - to initiate the first validator
+election. This election will transition the network from PoA into its second phase,
+[Nominated Proof of Stake (NPoS)](learn-staking), where the network is
+[secured by the economic stake](learn-security) that is bonded to the validators by the
+[nominators](maintain-nominator) and the validators themselves.
+
+Time-wise, the transition to Proof-of-Stake will largely depend on when a sufficient number of
+validators have registered and are ready to take over the security of the network. This number can
+be as low as 50 but probably closer to 100, as it was with [Kusama](https://kusama.network).
+
+This phase will last for at least two months.
```

Comment:
Why?

Reply:
It took a month on kusama IIRC so just doubling that. You don't think so?

Reply:
I think it could happen more quickly this time around since there's already a large validator community built on Kusama, many of whom will be ready to set up similar infrastructure for Polkadot.

Reply:
I'm fairly confident more than 100 will signal in the first week already, but I think Gav will want to play it safe and not jump on that signal. You're right though, it's unrealistic to estimate *anything* in this phase, I'll remove this part.